hs_core/templates/pages/genericresource.html

```diff
@@ -744,6 +747,11 @@ <h4 class="modal-title">Confirm files deletion</h4>
                                                             title="Unzip here">
                                                         <span class="fa fa-file-archive-o icon-blue"></span>
                                                     </button>
+                                                    <button id="fb-unzip-to-folder" data-fb-action="unzip-to-folder"
+                                                            class="disabled fb-space-right"
```

Comment:
for me, this "unzip to folder" option shows in the dropdown even when right click on a FOLDER (not a zip)
![image](https://user-images.githubusercontent.com/17934193/191293413-957ccbbe-77ad-457b-bc09-654674981121.png)

Reply:
your commit to fix this looks good